index.html

```diff
@@ -0,0 +1,330 @@
+              <img src="images/work2.jpg" alt="Unplugged Retreat" />
+            </div>
+            <div class="portfolio-content">
+              <h2>Multi Post Stories</h2>
+              <ul class="list-items">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+          <div class="portfolio-card">
+            <div class="portfolio-thumbnail">
+              <img src="images/work3.jpg" alt="Unplugged Retreat" />
+            </div>
+            <div class="portfolio-content">
+              <h2>Tonic</h2>
+              <ul class="list-items">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+          <div class="portfolio-card">
+            <div class="portfolio-thumbnail">
+              <img src="images/work4.jpg" alt="Unplugged Retreat" />
+            </div>
+            <div class="portfolio-content">
+              <h2>Multi Post Stories</h2>
+              <ul class="list-items">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </section>
+      <!-- End portfolio section -->
+      <!-- Start About section -->
+      <section id="about">
+        <div class="container">
+          <div class="about-content">
+            <div class="about-main-content">
+              <h2>About Myself</h2>
+              <p>
+                I'm a software developer! I can help you build a project,
+                feature or website look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.
+              </p>
```

Comment:
- [ ] I am not able to access the stylings of the paragraph since there was an inheritance of colors. As the previous reviewer suggested, kindly stick to the Figma design as much as possible. This is particularly on the colors. The colors mostly used in this paragraph, include `#344563`. Please modify the color to match the Figma design. This will make our project more professional.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/228250942-049b6e90-cf82-43a9-bf35-99685b50c82d.png)

Reply:
Can you kindly elaborate? I don't quite understand you yet.

Reply:
> Can you kindly elaborate? I don't quite understand you yet.

Dear @YugynDprodigy10 , I was suggesting that you change the color of the text to match the Figma design. This is your current color `#396df2` which does not match the Figma design. This same changes were requested by first reviewer and had not yet been applied.